lib/turbograft.rb

```diff
@@ -36,7 +36,7 @@ def referer
 
       ActiveSupport.on_load(:action_view) do
         (ActionView::RoutingUrlFor rescue ActionView::Helpers::UrlHelper).module_eval do
```

Comment:
do you actually need to `module_eval` anything here? AFAIR `Module#include` and `Module#prepend` were made public in v2.1.